lib/grape/cookies.rb

```diff
@@ -32,7 +32,7 @@ def each(&block)
     end
 
     def delete(name, opts = {})
-      options = opts.merge(value: 'deleted', expires: Time.at(0))
+      options = opts.merge!(value: 'deleted', expires: Time.at(0))
```

Comment:
😯 I thought the arguments were copied, but they are the same to those which are passed to the method. Very nice explanation 👏.